src/components/side_panel/chart/building_blocks/general_design/general_design_editor.xml

```diff
@@ -0,0 +1,30 @@
+<templates>
+  <t t-name="o-spreadsheet-GeneralDesignEditor">
+    <t t-set="chart_title">Chart Title</t>
+    <SidePanelCollapsible collapsedAtInit="false" name="'general'" class="general_collapsible">
```

Comment:
replace the manual name by a generated uuid in the component `SidePanelCollapsible` itself